app/src/main/kotlin/io/privkey/keep/storage/AndroidKeystoreStorage.kt

```
@@ -715,4 +764,146 @@ class AndroidKeystoreStorage(
+            }
+        }.getOrElse { e -> throw cipherInitFailure(e, "decryption") }
+    }
+
+    @Synchronized
+    private fun deleteDkgSecret() {
+        dkgSecretPrefs.edit().clear().commit()
+        try {
+            if (keyStore.containsAlias(DKG_SECRET_ALIAS)) {
+                keyStore.deleteEntry(DKG_SECRET_ALIAS)
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)